src/kfactory/kcell.py

```diff
@@ -2759,6 +2767,9 @@ def wrapped_cell(
                     return cell
 
                 _cell = wrapped_cell(**params)
+                _cell.info.update(info or {})
+                for pp in post_process or []:
```

Comment:
post_process i never None, therefore needs no check.

```suggestion
                if info is not None:
                    _cell.info.update(info)
                for pp in post_process:
```